pkg/apis/application/v1alpha1/types.go

```diff
@@ -1725,7 +1725,7 @@ func (s *SyncWindows) active(currentTime time.Time) *SyncWindows {
 
 	if s.HasWindows() {
 		var active SyncWindows
-		specParser := cron.NewParser(cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow)
+		specParser := cron.NewParser(cron.SecondOptional | cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow)
```

Comment:
Looking at it again, I don't think this change is actually necessary. Since the parser format was previously hard-coded, I think it's safe to keep the old hard-coded format. This line won't be affected by the change in default format.